traffic_portal/app/src/common/modules/header/header.tpl.html

```diff
@@ -40,7 +40,7 @@
             <li role="presentation" class="dropdown" ng-if="hasCapability('change-logs-read')">
                 <div class="btn-group" title="Change Logs" uib-dropdown is-open="alerts.isopen">
                     <button id="alertsButton" type="button" class="btn btn-link" ng-click="getChangeLogs()" uib-dropdown-toggle>
-                        <i class="fa fa-comment-o"></i>
+                        <i class="fa" ng-class="{ 'fa-comment': newLogCount() > 0, 'fa-comment-o': newLogCount() == 0 }"></i>
```

Comment:
`ng-class` supports expresions that evaluate to strings and arrays of strings as well as object property mappings. What I'm getting at is I think it's simpler - and calls `newLogCount` less times - to use `ng-class="{ newLogCount() > 0 ? 'fa-comment' : 'fa-comment-o'}"`

Reply:
yep, that's better

Reply:
unfortunately this syntax is not welcome so i'll just leave it as is

```
<i class="fa" ng-class="{ newLogCount() &gt 0 ? 'fa-comment' : 'fa-comment-o'}"></i>
```

Reply:
Oh, that's because it doesn't need the curly braces. It's not an object anymore. My bad.